modules/atoms/Heading.module.css

```
@@ -39,4 +39,5 @@
   color: var(--font-color-strong);
   font-size: 1.5rem;
   margin-bottom: 1rem;
+  margin-top: 2rem;
```

I did not notice that the h3 have the same font-size than h2, they should be smaller. You can reduce the margin-bottom to make it more consistent with the font-size

```suggestion
  font-size: 1.2rem;
  margin-bottom: 0.3rem;
  margin-top: 2rem;
```